index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html>
+	<meta charset="UTF-8">
```

Comment:
Are you using 4 for tab space? I think that is better if you use tab space 2, just to keep a convention.